_includes/macros/mobile-carousel.html

```diff
@@ -0,0 +1,50 @@
+
+{# ==========================================================================
+
+   mobile_carousel.render()
+
+   ==========================================================================
+
+   Description:
+
+   Render HTML for a carousel of slides with an image, title, and description.
+   Argument is:
+
+   slides: An array of data to populate the carousel slides with.
+           The structure looks like:
+           {
+           'image_sizes_class': <A image sizing class for responsive sizes>,
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/103ef847f88472457936689fc4013cfaef2ab7f8